content/_index.html

```diff
@@ -0,0 +1,495 @@
+        <div class="columns">
+          <div class="column is-half">
+            <h2 class="has-text-white"><i class="fa-light fa-download"></i> Quick Local Install </h2>
+            <p>RAPIDS offers several installation methods, the quickest is shown below. For more detailed guides go to
+              our <a href="https://docs.rapids.ai/install">advanced install page <i
+                  class="fa-solid fa-arrow-up-right"></i></a> </p>
+
+            <h3 class="mt-5 has-text-white">Requirements</h3>
+            <p><strong>A.</strong> NVIDIA Pascal™ or better GPU with a <a href="https://developer.nvidia.com/cuda-gpus"
+                taget="_blank">compute capability 6.0 and above <i class="fa-solid fa-arrow-up-right"></i></a></p>
+            <p><strong>B.</strong> Ubuntu 20.04, CentOS, Rocky Linux 8 or via WSL2 on Windows 11</p>
+            <p><strong>C.</strong> Recent <a href="https://docs.nvidia.com/deploy/cuda-compatibility/index.html"
+                target="_blank"> CUDA-version <em>and</em> NVIDIA-driver pairs</a>. Check yours with:
+              <code>nvidia-smi</code>
+
+            <h3 class="mt-6 has-text-white">Install with Conda</h3>
+            <p><strong>1.</strong> Download and run the conda install script: </p>
+            <pre class="highlight use-white-copy"><code>wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash Miniconda3-latest-Linux-x86_64.sh</code></pre>
+            <br>
+            <p><strong>2.</strong> Then quick install rapids with:</p>
+            <pre
+              class="highlight use-white-copy"><code>conda create -n rapids-23.02 -c rapidsai -c conda-forge -c nvidia  \ rapids=23.02 python=3.10 cudatoolkit=11.8</code></pre>
+
+            <h3 class="mt-6 has-text-white">Install with PIP</h3>
+            <p><strong>1.</strong> Install via pip channels:</p>
+            <pre class="highlight use-white-copy"><code>pip install cudf-cu11 dask-cudf-cu11 --extra-index-url=https://pypi.nvidia.com 
+pip install cuml-cu11 --extra-index-url=https://pypi.nvidia.com
+pip install cugraph-cu11 --extra-index-url=https://pypi.nvidia.com</code></pre>
+
+            <h3 class="mt-6 has-text-white">Install with Docker</h3>
+            <p>Use the <a href="https://docs.rapids.ai/install#docker">install page selector <i
+                  class="fa-solid fa-arrow-up-right"></i> </a></p>
+
+            <h3 class="mt-6 has-text-white">Install on Windows</h3>
+            <p>Use the Windows <a href="https://docs.rapids.ai/install#WSL2">WSL2 installation instructions <i
+                  class="fa-solid fa-arrow-up-right"></i> </a></p>
+
+            <h2 class="mt-8 has-text-white"><i class="fa-light fa-gears"></i> Do I need an Advanced Install?</h2>
+            <p> If you want to use Docker, WSL2, have an uncommon OS, hardware configuration, environment, or
+              need specific versioning such as <code>CUDA 11.6</code> or <code>Python 3.8</code> then you should use the
```

Comment:
We don't officially support 11.6, but version 11.5 is in the release selector.
```suggestion
              need specific versions such as <code>CUDA 11.5</code> or <code>Python 3.8</code> then you should use the
```